R/cdfPlot.R

```diff
@@ -146,110 +252,97 @@ cdfPlot <- function(contrastDF,
     # Combo PLOT: full data inset, most significant data in main plot
     # Rank by p-value
     contrastDF <- contrastDF %>%
-        dplyr::arrange(!!sym(y))
+        dplyr::arrange(!!sym(pvalCol))
     contrastDF$Rank <- c(1:nrow(contrastDF))
 
-    # Let's plot the p-value subsets
-    Sig <- contrastDF[[y]] <= pThreshold
-    NotSig <- contrastDF[[y]] > pThreshold
-
-    # Create group factor column in contrastDF
+    # Let"s plot the p-value subsets
     contrastDF$group <- NA
-    contrastDF$group[Sig] <- "Significant"
-    contrastDF$group[NotSig] <- "Not Significant"
-
-    contrastDF$group <- contrastDF$group %>%
-        factor(levels = groupNames)
-
-    # Set an order field to force plotting of 'not significant' first
     contrastDF$order <- NA
-    contrastDF$order[NotSig] <- 1
-    contrastDF$order[Sig] <- 2
+    contrastDF <- contrastDF %>%
+        dplyr::mutate(group = dplyr::case_when(!!rlang::sym(pvalCol) <= pThreshold ~ "Significant",
+                                               !!rlang::sym(pvalCol) > pThreshold ~ "Not Significant"),
+                      group = factor(group,
+                                     levels = c("Significant", "Not Significant")),
+                      order = dplyr::case_when(group == "Not Significant" ~ 1,
```

Comment:
This column is not used anywhere. I think it was used for ggplot, but not any mote